app/server/middleware/graphql/validateRecord.ts

```diff
@@ -3,6 +3,11 @@ import Ajv, { ErrorObject } from "ajv";
 const ajv = new Ajv({ allErrors: true });
 // AJV needs to be made aware of any custom formats used in the schema
 ajv.addFormat("rfpDigits", /\d{3,4}/);
+ajv.addFormat("email", /^[\.\w-]+@([\w-]+\.)+[\w-]{2,4}$/);
+ajv.addFormat(
+  "phone",
+  /^(\+?\d{1,2}[\s,-]?)?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4}$/
+);
```

Comment:
This file should import from our `app/data/jsonSchemaForm/customFormats.ts` file instead of duplicating the regexes - this might be a good 'first issue' refactor, I'll create a card